### PR TITLE
Add support for sdk 0.11.x

### PR DESCRIPTION
**Pull request by galak**
Pull in changes from PR #20221 for just adding in the files associated with SDK 0.11.x support.  For now we add in a CI test change to see how things work with the new toolchain.

### CODE REVIEW

File: cmake/toolchain/zephyr/0.11/target.cmake [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# SPDX-License-Identifier: Apache-2.0
+
+set(CROSS_COMPILE_TARGET_arm         arm-zephyr-eabi)
```
Comment (stephanosio) [inline cmake/toolchain/zephyr/0.11/target.cmake]: We need to do something like below to get AArch64 target working by the way:
```
if(CONFIG_64BIT)
  set(CROSS_COMPILE_TARGET_arm  aarch64-zephyr-elf)
else()
  set(CROSS_COMPILE_TARGET_arm  arm-zephyr-eabi)
endif()
```

Comment (galak) [reply cmake/toolchain/zephyr/0.11/target.cmake]: Should be added in the aarch64 PR.

**Review by nashif — APPROVED** [review]

**Comment by galak** [comment]
Approved by TSC for merge into 2.1

File: cmake/toolchain/zephyr/0.11/target.cmake (hunk 1 above)
Comment (stephanosio) [reply cmake/toolchain/zephyr/0.11/target.cmake]: cc @carlocaione

Comment (carlocaione) [reply cmake/toolchain/zephyr/0.11/target.cmake]: ACK

**Pull request merged**